vmdb/spec/requests/api/policies_spec.rb

```diff
@@ -28,32 +28,29 @@
 
   include Rack::Test::Methods
 
-  before(:each) do
-    init_api_spec_env
-
-    @zone       = FactoryGirl.create(:zone, :name => "api_zone")
-    @miq_server = FactoryGirl.create(:miq_server, :guid => miq_server_guid, :zone => @zone)
-    @ems        = FactoryGirl.create(:ems_vmware, :zone => @zone)
-    @host       = FactoryGirl.create(:host)
+  let(:zone)        { FactoryGirl.create(:zone, :name => "api_zone") }
+  let(:miq_server)  { FactoryGirl.create(:miq_server, :guid => miq_server_guid, :zone => zone) }
+  let(:ems)         { FactoryGirl.create(:ems_vmware, :zone => zone) }
+  let(:host)        { FactoryGirl.create(:host) }
 
-    Host.any_instance.stub(:miq_proxy).and_return(@miq_server)
-
-    # Creating:  policy_set_1 = [policy_1, policy_2]  and  policy_set_2 = [policy_3]
+  let(:p1)          { FactoryGirl.create(:miq_policy, :description => "Policy 1") }
+  let(:p2)          { FactoryGirl.create(:miq_policy, :description => "Policy 2") }
+  let(:p3)          { FactoryGirl.create(:miq_policy, :description => "Policy 3") }
 
-    @p1  = FactoryGirl.create(:miq_policy, :description => "Policy 1")
-    @p2  = FactoryGirl.create(:miq_policy, :description => "Policy 2")
-    @p3  = FactoryGirl.create(:miq_policy, :description => "Policy 3")
+  let(:ps1)         { FactoryGirl.create(:miq_policy_set, :description => "Policy Set 1") }
+  let(:ps2)         { FactoryGirl.create(:miq_policy_set, :description => "Policy Set 2") }
 
-    @ps1 = FactoryGirl.create(:miq_policy_set, :description => "Policy Set 1")
-    @ps2 = FactoryGirl.create(:miq_policy_set, :description => "Policy Set 2")
+  let(:p_guids)     { [p1.guid, p2.guid] }
```

Comment:
kinda want to say just include this inline below in the 3 places too. (the count can just be hardcoded to 3)

no reason to "DRY" this in the tests. explicit is better